gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```

Comment:
Any changes between 7.4 and 7.6?

Reply:
just added support for java 18 and 19, and some other dependency format supports, nothings been removed so shouldn't affect our current code